main.go

```
@@ -1,1051 +1,1338 @@
+			domainVal = ""
+			pathVal = routeKey
+		case "domain", "wildcard", "regex":
+			domainVal = routeKey
+			pathVal = ""
+		default:
+			log.Warn().Str("route_key", routeKey).Str("type", route.Type).Msg("Unknown route type; skipping")
+			routesFailed++
+			continue
+		}
+
+		_, err := db.Exec(
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)